meta/meta.go

```diff
@@ -1040,23 +1040,28 @@ func (m *Meta) GetLastNHistoryDDLJobs(num int) ([]*model.Job, error) {
 }
 
 // LastJobIterator is the iterator for gets latest history.
-type LastJobIterator struct {
-	iter *structure.ReverseHashIterator
+type LastJobIterator interface {
+	GetLastJobs(num int, jobs []*model.Job) ([]*model.Job, error)
 }
 
 // GetLastHistoryDDLJobsIterator gets latest N history ddl jobs iterator.
-func (m *Meta) GetLastHistoryDDLJobsIterator() (*LastJobIterator, error) {
+func (m *Meta) GetLastHistoryDDLJobsIterator() (LastJobIterator, error) {
 	iter, err := structure.NewHashReverseIter(m.txn, mDDLJobHistoryKey)
 	if err != nil {
 		return nil, err
 	}
-	return &LastJobIterator{
+	return &HLastJobIterator{
 		iter: iter,
 	}, nil
 }
 
+// HLastJobIterator is the iterator for gets the latest history.
+type HLastJobIterator struct {
```

Comment:
Why add this struct or make this change？

Reply:
we will add another implementation(concurrent ddl implementation) for `LastJobIterator` in the future